rust/operator-binary/src/controller.rs

```diff
@@ -104,12 +104,10 @@ const DEFAULT_CONSOLE_LOG_LEVEL: LogLevel = LogLevel::INFO;
 const DEFAULT_SERVER_LOG_LEVEL: LogLevel = LogLevel::INFO;
 const DEFAULT_DECISION_LOG_LEVEL: LogLevel = LogLevel::NONE;
 
-// bundle builder: ~ 5 MB x 2
-// these sizes are needed both for the single file (for multilog, in bytes) as well as the total (for the EmptyDir)
+// bundle builder: ~ 5 MB x 5
+// these sizes are needed both for the single file (for rotation, in bytes) as well as the total (for the EmptyDir)
 const OPA_ROLLING_BUNDLE_BUILDER_LOG_FILE_SIZE_MB: u32 = 5;
```

Comment:
Can you please comment this variable out, so that no one get's a false assumption about the rotation?
Additionally, please place a quick disclaimer like "ideally we would rotate by size, but we currently rotate every hour,  https://github.com/stackabletech/opa-operator/issues/606 will fix this"

Reply:
edbe052e3464ef485e6d5743d177f32c73f88700

Can't remove it entirely since it's still used for the volume size calculation right below.

Reply:
That makes sense, thx!